backend/infrahub/core/schema/basenode_schema.py

```diff
@@ -490,7 +491,73 @@ def get_uniqueness_constraint_type(
             return UniquenessConstraintType.SUBSET_OF_HFID
         return UniquenessConstraintType.STANDARD
 
+    def _update_element_list(
+        self, element_list: list[str] | None, name_update_map: dict[str, str], deleted_field_names: set[str]
```

Comment:
typing should be `element_list: list[str]` as we already check that inputs are not None while using this